frame/contracts/src/weights.rs

```diff
@@ -150,1181 +150,1183 @@ pub trait WeightInfo {
+			.saturating_add(RocksDbWeight::get().reads(6 as Weight))
 			.saturating_add(RocksDbWeight::get().writes(2 as Weight))
 	}
 	fn seal_clear_storage(r: u32, ) -> Weight {
 		(0 as Weight)
-			// Standard Error: 2_154_000
-			.saturating_add((5_341_117_000 as Weight).saturating_mul(r as Weight))
-			.saturating_add(RocksDbWeight::get().reads(4 as Weight))
+			// Standard Error: 2_632_000
+			.saturating_add((2_148_012_000 as Weight).saturating_mul(r as Weight))
+			.saturating_add(RocksDbWeight::get().reads(5 as Weight))
 			.saturating_add(RocksDbWeight::get().reads((100 as Weight).saturating_mul(r as Weight)))
 			.saturating_add(RocksDbWeight::get().writes(1 as Weight))
 			.saturating_add(RocksDbWeight::get().writes((100 as Weight).saturating_mul(r as Weight)))
 	}
 	fn seal_get_storage(r: u32, ) -> Weight {
-		(62_353_000 as Weight)
-			// Standard Error: 1_183_000
-			.saturating_add((1_141_653_000 as Weight).saturating_mul(r as Weight))
-			.saturating_add(RocksDbWeight::get().reads(4 as Weight))
+		(48_127_000 as Weight)
+			// Standard Error: 1_123_000
+			.saturating_add((906_947_000 as Weight).saturating_mul(r as Weight))
+			.saturating_add(RocksDbWeight::get().reads(5 as Weight))
 			.saturating_add(RocksDbWeight::get().reads((100 as Weight).saturating_mul(r as Weight)))
 	}
 	fn seal_get_storage_per_kb(n: u32, ) -> Weight {
-		(905_905_000 as Weight)
-			// Standard Error: 363_000
-			.saturating_add((155_161_000 as Weight).saturating_mul(n as Weight))
-			.saturating_add(RocksDbWeight::get().reads(5 as Weight))
+		(676_986_000 as Weight)
+			// Standard Error: 307_000
+			.saturating_add((153_667_000 as Weight).saturating_mul(n as Weight))
+			.saturating_add(RocksDbWeight::get().reads(6 as Weight))
 	}
 	fn seal_transfer(r: u32, ) -> Weight {
-		(60_519_000 as Weight)
-			// Standard Error: 1_942_000
-			.saturating_add((6_453_551_000 as Weight).saturating_mul(r as Weight))
+		(36_730_000 as Weight)
+			// Standard Error: 1_966_000
+			.saturating_add((3_972_101_000 as Weight).saturating_mul(r as Weight))
 			.saturating_add(RocksDbWeight::get().reads(5 as Weight))
 			.saturating_add(RocksDbWeight::get().reads((100 as Weight).saturating_mul(r as Weight)))
 			.saturating_add(RocksDbWeight::get().writes(1 as Weight))
 			.saturating_add(RocksDbWeight::get().writes((100 as Weight).saturating_mul(r as Weight)))
 	}
 	fn seal_call(r: u32, ) -> Weight {
-		(192_122_000 as Weight)
-			// Standard Error: 7_851_000
-			.saturating_add((10_736_771_000 as Weight).saturating_mul(r as Weight))
-			.saturating_add(RocksDbWeight::get().reads(5 as Weight))
-			.saturating_add(RocksDbWeight::get().reads((100 as Weight).saturating_mul(r as Weight)))
+		(0 as Weight)
+			// Standard Error: 10_776_000
+			.saturating_add((9_860_978_000 as Weight).saturating_mul(r as Weight))
+			.saturating_add(RocksDbWeight::get().reads(6 as Weight))
+			.saturating_add(RocksDbWeight::get().reads((200 as Weight).saturating_mul(r as Weight)))
```

Comment:
there is quite some changes in the number of read, do you know why ?

Reply:
It means we have now 2 reads instead of 1. It is multiplied by 100 because that is the batch size. If you look at the other benchmark results you see that we have an additional read for almost every other function. I think I must have added this in a previous PR where I did not run the benchmarks for. Not sure which could have caused this.

Reply:
I see ok

Reply:
It would be nice to know, though. I look into this.

Reply:
maybe we can make a small PR just to fix the benchmark just to confirm,
But the logic of this PR is fine otherwise